### ISSUE TITLE
Friendly URL support

### ISSUE THREAD

**Issue by NareshBisht** [LazZiya/TagHelpers#21]
I want to request the following feature.

**1st feature:**

The problem I'm facing is that right now the URL generated is like this "localhost:44339/?p=2&s=10". I want to remove "s=10" from the query and make the URL like "localhost:44339/page/2".

**2nd feature:**

The existing URL query of the page is always added first and then the page-number(p=2&s=10) query is added which in my case breaks the page routing can you provide an option so that the user gets to decide whether the page number query comes first or at last. For example, the "Paging TagHelper" for a page with the URL "www.test.com?search=cat&sort=name" creates href like "www.test.com?search=cat&sort=name&p=2&s=10" but I want the href to be like "www.test.com?p=2&s=10&search=cat&sort=name".

**Comment by LazZiya** [comment]
Hi @NareshBisht 

Well noted, I will work to include your request in the next update.

Best,
Ziya

**Comment by LazZiya** [comment]
Long time for this issue :) Just fixed with [v6.0.2](https://github.com/LazZiya/TagHelpers/releases/tag/v6.0.2) available in nuget.

Now you can define custom URL template:

````html
<paging total-records="Model.TotalRecords" 
        page-no="Model.PageNo" 
        page-size="Model.PageSize" 
        url-template="{0}-{1}">
</paging>
````

See the [basic setup docs](https://docs.ziyad.info/en/LazZiya.TagHelpers/v6.0/Paging-TagHelper-Basic-Setup.md) for more details.
